src/pages/index.js

```diff
@@ -71,10 +71,22 @@ export async function getStaticProps({ preview, previewData }) {
   const { children } = res;
 
   const blocks = formatBlocksForSections(props?.post?.blocks);
+
+  const { configuration } = await fetchJson(
+    `${process.env.HURUMAP_API_URL}profile_by_url/?format=json`
+  );
+  const featuredCounties = configuration?.featured_geographies?.county;
```

Comment:
@kilemensi yes, they are just label for geography. We can add any

Reply:
Cool, then probably not for this PR @KhadijaMahanga  but lets think of a way of using `level1`, `level2`, ... `levelN` moving forward so that we can generalize our functions and make them reusable across multiple countries with different levels.